src/gui/pages_menu/KM_GUIMenuLobby.pas

```diff
@@ -396,6 +396,7 @@ procedure TKMMenuLobby.CreateControls(aParent: TKMPanel);
         DropBox_LobbyPlayerSlot[I] := TKMDropColumns.Create(Panel_LobbyPlayers, C1, OffY, 150, 20, fnt_Grey, '', bsMenu, False);
         DropBox_LobbyPlayerSlot[I].DropWidth := DropWidth;
         DropBox_LobbyPlayerSlot[I].SetColumns(fnt_Outline, ['', gResTexts[TX_MENU_MAP_TITLE]], [0, 100 + Max(0, 40 - TxtWidth)], [True, False]);
+        DropBox_LobbyPlayerSlot[I].List.Columns[1].TriggerOnChange := False; //1st column should not trigger OnChange event
```

Comment:
Please convert this comment from "what are we doing" into "why are we doing this". Why is it that the 1st column should not trigger OnChange?